git_pki/utils.py

```diff
@@ -8,6 +8,11 @@
 
 from git_pki.custom_types import ShellCommand
 
+def is_string(value) -> bool:
+    try:
+        return isinstance(value, basestring)
+    except NameError:
+        return isinstance(value, str)
```

Comment:
```suggestion
def is_string(value) -> bool:
    return isinstance(value, str)
```
Since we don't support python 2.X, comparision to `basestring` is redundant, or am I missing something?